frontend/src/static/js/components/webstatus-stats-missing-one-impl-chart-panel.ts

```diff
@@ -0,0 +1,153 @@
+      task: async ([apiClient, startDate, endDate]: [
+        APIClient,
+        Date,
+        Date,
+      ]) => {
+        await this._fetchMissingOneImplemenationCounts(
+          apiClient,
+          startDate,
+          endDate,
+        );
+        return;
+      },
+    });
+  }
+
+  getDisplayDataChartOptionsInput(): {
+    seriesColors: string[];
+    vAxisTitle: string;
+  } {
+    // Compute seriesColors from selected browsers and BROWSER_ID_TO_COLOR
+    const selectedBrowsers = this.supportedBrowsers;
+    const seriesColors = [...selectedBrowsers].map(browser => {
+      const browserKey = browser as keyof typeof BROWSER_ID_TO_COLOR;
+      return BROWSER_ID_TO_COLOR[browserKey];
+    });
+
+    return {
+      seriesColors: seriesColors,
+      vAxisTitle: 'Number of features missing',
+    };
+  }
+
+  async _fetchMissingOneImplemenationCounts(
+    apiClient: APIClient,
+    startDate: Date,
+    endDate: Date,
+  ) {
+    if (typeof apiClient !== 'object') return;
+    const browserMetricData: Array<
+      LineChartMetricData<BrowserReleaseFeatureMetric>
+    > = ALL_BROWSERS.map(browser => ({
+      label: browser,
+      data: [],
+      getTimestamp: (item: BrowserReleaseFeatureMetric) =>
+        new Date(item.timestamp),
+      getData: (item: BrowserReleaseFeatureMetric) => item.count,
+    }));
+    const promises = ALL_BROWSERS.map(async browser => {
+      const browserData = browserMetricData.find(
+        data => data.label === browser,
+      );
+      if (!browserData) return;
+
+      const otherBrowsers = ALL_BROWSERS.filter(value => browser !== value);
+      for await (const page of apiClient.getMissingOneImplementationCountsForBrowser(
+        browser,
+        otherBrowsers,
+        startDate,
+        endDate,
+      )) {
+        browserData.data.push(...page);
+      }
+    });
+    await Promise.all(promises); // Wait for all browsers to finish
+
+    this.setDisplayDataFromMap(browserMetricData);
+  }
+  getPanelID(): string {
+    return 'missing-one-implementation-chart';
+  }
+  getPanelText(): string {
+    return 'Features missing in only 1 browser';
+  }
+  renderControls(): TemplateResult {
+    return html`
+      <sl-dropdown
+        id="${this.getPanelID()}-browser-selector"
+        multiple
+        stay-open-on-select
+        .value="${this.supportedBrowsers.join(' ')}"
+      >
+        <sl-button slot="trigger">
+          <sl-icon slot="suffix" name="chevron-down"></sl-icon>
+          Browsers
+        </sl-button>
+        <sl-menu @sl-select=${this.handleBrowserSelection}>
+          <sl-menu-item type="checkbox" value="chrome">Chrome</sl-menu-item>
+          <sl-menu-item type="checkbox" value="edge">Edge</sl-menu-item>
+          <sl-menu-item type="checkbox" value="firefox">Firefox</sl-menu-item>
+          <sl-menu-item type="checkbox" value="safari">Safari</sl-menu-item>
+        </sl-menu>
+      </sl-dropdown>
+    `;
+  }
+
+  handleBrowserSelection(event: Event) {
```

Comment:
Could this also be moved into the base class?

Reply:
Definitely. I have that as a future change for #1104. I only didn't move it yet because I am wondering how it will work then. (maybe each chart will be slightly different?). I will put a TODO comment though so I won't forget.